### DISCUSSION TITLE
[Projects Beta] Ability to rename the `No Status` value in fields / board columns

### DISCUSSION THREAD

**Discussion by thclark** [community/community#8261] [Projects and Issues]
I have status fields that look like this:
![Screenshot 2021-11-30 at 10 50 25](https://user-images.githubusercontent.com/7223170/144034894-b747bb28-7ecb-4f32-9ad6-ecebed83f732.png)

Where the triage column has an obvious purpose (unassigned statuses on issues created by my devs get chucked in there for me to triage and prioritise).

However, the default value `No Status` has a column on the board that I can't delete, and can't edit the name of, which is redundant with Triage:
![Screenshot 2021-11-30 at 10 50 48](https://user-images.githubusercontent.com/7223170/144034881-86b13b5e-2229-4abb-9426-340cc3b5636e.png)

My workaround is to simply treat `No Status` as untriaged but I think it'd be good to be able to name all the columns for what they actually are.

**Comment by createdbyjurandatbayer** [comment]
I would also like to change "No Status" to "Backlog" :)

**Comment by BlockchainSpaceDev** [comment]
How did it go? Was it considered? It's pretty annoying not to rename it since in a Backlog this status has no meaning
